### ISSUE TITLE
asdkh

### ISSUE THREAD

**Issue by hizeci** [edge-bird/edge-bird.github.io#1]
Why your blog loads ssssssssssssssssssssssssoooooooooooooo sssssssssssssssslllllllllllllllllllooooooooooooooooowwwwwwwwwwllllllllllllyyyyyyyyyyyyyyyyyyyy!!!!!!!!!!

And please remove the link about me

**Comment by edge-bird** [comment]
no, I refused your request because I don't people to gua on the "link", and nobody will see it, you can put your heart right now, and now, My blog loads so fffffffffffaaaaaaaaaaaaasssssssssssssttttttttttttt, by the way, now, you are Scene ?


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "edge-bird/edge-bird.github.io"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年3月19日(星期五) 下午3:00
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[edge-bird/edge-bird.github.io] asdkh (#1)





 
Why your blog loads ssssssssssssssssssssssssoooooooooooooo sssssssssssssssslllllllllllllllllllooooooooooooooooowwwwwwwwwwllllllllllllyyyyyyyyyyyyyyyyyyyy!!!!!!!!!!
 
And please remove the link about me

**Comment by twt-tec** [comment]
I am the offical account of twt-tec, I recommend you make the link of [twt-tec's offical website](https://twt-tec.github.io/) larger! NOW!
